index.html

```diff
@@ -10,8 +10,98 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <!-- Top Left Karma Logo -->
+    <header>
+        
+            <img src="img\karma-logo.svg" 
+            alt="Karma Logo" 
+            class="header__logo"/>
+        
+    <!-- Top Right Navigation Links-->
+    <nav>
+        <ul class="navigation__list">
+            <li class="navigation__item">
+                <a class="navigation__link" style="color:darkslategrey" href="#">Meet Karma</a>
+            </li>
+            <li class="navigation__item">
+                <a class="navigation__link" href="#">How It Works</a>
+            </li>
+            <li class="navigation__item">
+                <a class="navigation__link" href="#">Store</a>
+            </li>
+            <li class="navigation__item">
+                <a class="navigation__link" href="#">Blog</a>
+            </li>
+            <li class="navigation__item">
+                <a class="navigation__link" href="#">Help</a>
+            <li class="navigation__item">
+                <a class="navigation__link" href="#">Login</a>
+                </li>
+            </li>
+        </ul>
+    </nav>
+    </header>
+
+    <!-- Main -->
+    <main class="main_image"> 
+        <h1 class="main_intro">Introducing Karma</h1>
+        <h2 class="wifi_promo">
+            Bring WiFi with you, everywhere you go.
+        </h2>
+        <button class="learn_more_button">Learn More</button>
+    </main>
```

Comment:
The `main` should contain everything that is not the `header` and the `footer` - not just the top hero section.